components/Auth/TokenAuthorization.js

```diff
@@ -78,7 +114,8 @@ class TokenAuthorization extends Component {
       echo,
       email,
       error,
-      loading
+      loading,
+      noAutoAuthorize
```

Comment:
What is this?

Reply:
this is a new query param, going to be sent in the updated signIn-Email's notice link: "if you didn't request this, deny it here". It ensures, that no autoAuthorize is started.